### DISCUSSION TITLE
Firefox refund

### DISCUSSION THREAD

**Discussion by ReoJee** [aclap-dev/video-downloadhelper#1002] [License & Purchase requests]
### What's your issue?

Hi

have purchased today, doesn't work where i wanted it to work.

please refund.

### The 10 first digits (only 10) of your license number or email address

e5f28f6260

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by Hpmg32** [comment]
What's your issue?
Hi

have purchased today, doesn't work where i wanted it to work.

please refund.

The 10 first digits (only 10) of your license number or email address
9f68576306

I confirm that I did not not paste my full license number
 I confirm

**Comment by jcfrog** [comment]
@ReoJee Hi,
You have been refunded. Should be available in a few business days.

**Comment by jcfrog** [comment]
@Hpmg32 Hi,
You have been refunded. Should be available in a few business days.
